pkg/eks/update.go

```diff
@@ -162,6 +162,31 @@ func (c *ClusterProvider) UpdateClusterVersion(cfg *api.ClusterConfig) (*awseks.
 	return output.Update, nil
 }
 
+// UpdateClusterTags calls eks.TagResource and tags the cluster
+func (c *ClusterProvider) UpdateClusterTags(clusterConfig *api.ClusterConfig) error {
+	tags := make(map[string]*string)
+	for key, value := range clusterConfig.Metadata.Tags {
+		tags[key] = &value
+	}
+	if err := c.RefreshClusterStatus(clusterConfig); err != nil {
+		return err
+	}
+	input := &awseks.TagResourceInput{
+		ResourceArn: c.Status.clusterInfo.cluster.Arn,
+		Tags:        tags,
```

Comment:
hope you dont mind if I chime in.

eksctl is having some string util method such as
https://github.com/weaveworks/eksctl/blob/master/pkg/utils/strings/strings.go#L28 
https://github.com/weaveworks/eksctl/blob/master/pkg/utils/strings/strings.go#L39
https://github.com/weaveworks/eksctl/blob/master/pkg/utils/strings/strings.go#L48